core/App/components/misc/CredentialCard11.tsx

```diff
@@ -253,6 +372,22 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({ credential, style =
               {overlay.metaOverlay?.name}
             </Text>
           </View>
+          {error && (
+            <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+              <Icon style={[styles.errorIcon]} name="close" size={30} />
+
+              <Text style={[styles.errorText]} testID={testIdWithKey('RevokedOrNotAvailable')} numberOfLines={1}>
+                {t('ProofRequest.NotAvailableInYourWallet')}
+              </Text>
+            </View>
+          )}
+          <FlatList
+            data={proofItems}
+            scrollEnabled={false}
+            renderItem={({ item }) => {
+              return renderCardAttribute(item as Attribute & Predicate)
+            }}
+          />
         </View>
         {primaryField && (
```

Comment:
I think it would be better to make use of a single Flatlist here even for the primary and secondary attributes. That way styling is the same for cards used in both credentials and proofs